### PR TITLE
WIP sending QR Code with Twilio message

### PR DESCRIPTION
**Pull request by sarath-yerra**

### CODE REVIEW

File: src/Modules/Registration/RegistrationContainer.js [hunk 1]
```diff
@@ -96,6 +96,10 @@ const RegistrationContainer = (props) => {
     return location.state? `Your reservation time is at ${location.state.event_slot.start_time} - ${location.state.event_slot.end_time} on ${location.state.event_date}. `: "";
   }
 
+  const getCodeURL = (identification_code) => {
+    return `Your QRCode for the Reservation http://localhost:5000/qrcode/${identification_code}  `;
```
Comment (maheshkurma) [inline src/Modules/Registration/RegistrationContainer.js:100]: above URL needs to be updated, to work in  production/beta

Comment (maheshkurma) [reply src/Modules/Registration/RegistrationContainer.js]: fixed in commit - https://github.com/midohiofoodbank/freshtrak-client/pull/232/commits/f21ee4ea6f67f760190fa5b24cd11b6c4b982f4d

Comment (sarath-yerra) [reply src/Modules/Registration/RegistrationContainer.js]: thank you

Comment (sarath-yerra) [reply src/Modules/Registration/RegistrationContainer.js]: we should also add qrcode in the url

Comment (maheshkurma) [reply src/Modules/Registration/RegistrationContainer.js]: @sarath-yerra updated

Comment (sarath-yerra) [reply src/Modules/Registration/RegistrationContainer.js]: @maheshkurma when you have a chance can you please review this PR.

Comment (maheshkurma) [reply src/Modules/Registration/RegistrationContainer.js]: lets fix `qr_code` issue which i see in master branch, before we merge any qr related code.
@sarath-yerra  issue is seen in below screenshot : 
Screen Shot 2021-01-15 at 9 24 12 AM

Comment (sarath-yerra) [reply src/Modules/Registration/RegistrationContainer.js]: This is fixed and merged to master

**Comment by sarath-yerra** [comment]
Me and @maheshkurma Tested the QR Feature

**Comment by sarath-yerra** [comment]
In Local Environment:
Sent from your Twilio trial account - You have successfully registered for FreshTrak,  Your confirmation code is XRMDCC.
 Your QRCode for the Reservation http://localhost:3000/qrcode/xrmdcc

We have added root in the environment

**Review by maheshkurma — APPROVED** [review]
code reviewed.

**Pull request merged**